src/librustdoc/html/static/themes/dark.css

```diff
@@ -337,6 +337,11 @@ pre.ignore:hover, .information:hover + pre.ignore {
 	border-color: transparent black transparent transparent;
 }
 
+.important-traits-tooltiptext {
+	background-color: #111 !important;
```

Comment:
Do you really need `!important`? Generally, just giving a more precise selector allows to override things without needing them.

Reply:
Yes, because we're just overriding the same declaration in the themeless version. I added a decl in the themeless version with the intent of having okayish defaults.

Reply:
Maybe remove it from the themeless version then? We always have themes after all...